transports/onion/Cargo.toml

```
@@ -0,0 +1,30 @@
+[package]
+name = "libp2p-onion"
+version = "0.1.0"
```

```suggestion
version = "0.1.0-alpha"
```

Like with https://github.com/libp2p/rust-libp2p/pull/2622 and https://github.com/libp2p/rust-libp2p/pull/2289 I suggest releasing this new crate as an `alpha` for now. Objections?

---

Sounds good to me.

---

:exploding_head: wow that was fast. Thanks @umgefahren.